Applications/Internet/Internet Explorer 7.0/Online/script.js

```diff
@@ -20,7 +20,8 @@ var installerImplementation = {
             .version(LATEST_STABLE_VERSION)
             .distribution("upstream")
             .architecture("x86")
-            .prefix("InternetExplorer7")
+            .prefix(this._name, this._wineDistribution, this._wineArchitecture, this._wineVersion)
+            .luna()
```

Comment:
why did you add luna?

Reply:
Because it was in the fix for #627

Reply:
It's not related to the fix. No need to change it here.